apps/settings/style/settings.css

```diff
@@ -462,38 +462,31 @@ section ul.operate-certificate {
  */
 
 /* three-state items: on|off|unknown */
-#menuItem-callWaiting .checkbox-label,
-#menuItem-callWaiting[data-state="on"] .checkbox-label,
-#menuItem-callWaiting[data-state="off"] .checkbox-label {
+#menuItem-callWaiting[data-state="on"] span,
+#menuItem-callWaiting[data-state="off"] span {
   display: inherit;
 }
 
-#menuItem-callWaiting .alert-label,
-#menuItem-callWaiting[data-state="on"] .alert-label,
-#menuItem-callWaiting[data-state="off"] .alert-label {
+#menuItem-callWaiting span.cw-alert-icon,
+#menuItem-callWaiting[data-state="on"] span.cw-alert-icon,
+#menuItem-callWaiting[data-state="off"] span.cw-alert-icon {
   display: none;
 }
 
-#menuItem-callWaiting[data-state="unknown"] .checkbox-label input ~ span:after {
-  display: none;
+#menuItem-callWaiting[data-state="unknown"] label {
+  pointer-events: none;
+  opacity: 0.6;
```

Comment:
Previously UI designer prefer set opacity as 0.3 for gaia-switch and so on, maybe we could follow it if it does not look very strange.